### DISCUSSION TITLE
抽卡概率是否需要更新？

### DISCUSSION THREAD

**Discussion by monsterxcn** [Arondight/Adachi-BOT#716] [Ideas]
最近想看一下抽卡概率是如何实现的，看到此项目的概率是根据 [B 站专栏](https://www.bilibili.com/read/cv10468091) 写的：

![image](https://user-images.githubusercontent.com/22407052/158291364-682ed4be-d294-407b-b47a-60768ff7b119.png)

https://github.com/Arondight/Adachi-BOT/blob/78a8f929542a4132fcaa330c96b084a44de3e3a3/src/plugins/gacha/data.js#L36-L64

但我根据作者在 B 站专栏下面的评论，去看了下 NGA 的帖子，他给的公式是：

![image](https://user-images.githubusercontent.com/22407052/158291519-052c4cf8-855e-4bd5-937f-b2fc9f438e9a.png)

我看到 NGA 帖子附件下面显示是 `在2022-01-16 00:14修改`，这跟 B 站专栏的公式哪个才是更新的呢？😂

**Comment by Arondight** [comment]
大概这个项目用的已经老了，按照他的说法应该是以 nga 的为主吧，如果有时间帮忙更一下吧

**Reply by monsterxcn** [reply]
🆗

**Reply by Arondight** [reply]
> 🆗

麻烦顺手更新一下注释（网址和日期）

**Reply by mark9804** [reply]
算了一下，更新后的会稍微高一点，逻辑没有改变，应该是后面数据多了修正了偏差
